src/components/ui/Pool/PoolDeposit.tsx

```
@@ -119,57 +138,95 @@ const PoolDeposit = ({
+  const handleZapTokenChange = (selectedAddress: string) => {
+    setZapTokenAddress(selectedAddress);
+    setZapAmount("0");
+    setSliderValue(50);
+  };
 
   const handleWithdraw = () => {
-    if(!isConnected) {
-      openConnectModal && openConnectModal()
-      return
+    if (!isConnected) {
+      openConnectModal && openConnectModal();
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)